plugins/inputs/dpdk/sample.conf

```diff
@@ -14,20 +14,37 @@
   # socket_access_timeout = "200ms"
 
   ## Enables telemetry data collection for selected device types.
-  ## Adding "ethdev" enables collection of telemetry from DPDK NICs
-  ## (stats, xstats, link_status).
-  ## Adding "rawdev" enables collection of telemetry from DPDK Raw Devices
-  ## (xstats).
+  ## Adding "ethdev" enables collection of telemetry from DPDK NICs (stats, xstats, link_status, info).
+  ## Adding "rawdev" enables collection of telemetry from DPDK Raw Devices (xstats).
   # device_types = ["ethdev"]
 
   ## List of custom, application-specific telemetry commands to query
   ## The list of available commands depend on the application deployed.
   ## Applications can register their own commands via telemetry library API
-  ## http://doc.dpdk.org/guides/prog_guide/telemetry_lib.html#registering-commands
+  ## https://doc.dpdk.org/guides/prog_guide/telemetry_lib.html#registering-commands
   ## For L3 Forwarding with Power Management Sample Application this could be:
   ##   additional_commands = ["/l3fwd-power/stats"]
   # additional_commands = []
 
+  ## List of plugin options.
+  ## Supported options:
+  ##  - "dpdk_in_memory" option enables reading for multiple sockets when a dpdk application is running with --in-memory option.
+  ##    When option is enabled plugin will try to find additional socket paths related to provided socket_path.
+  ##    Details: https://doc.dpdk.org/guides/howto/telemetry.html#connecting-to-different-dpdk-processes
+  # plugin_options = ["dpdk_in_memory"]
+
+  ## Specifies plugin behavior regarding unreachable socket (which might not have been initialized yet).
+  ## Available choices:
+  ##   - error: Telegraf will return an error during the startup and gather phases if socket is unreachable
+  ##   - ignore: Telegraf will ignore error regarding unreachable socket on both startup and gather
+  # unreachable_socket_behavior = "error"
+
+  ## List of metadata fields which will be added to every metric produced by the plugin.
+  ## Supported options:
+  ##  - "dpdk_pid" - exposes PID of DPDK process. Example: dpdk_pid=2179660i
+  ##  - "dpdk_version" - exposes version of DPDK. Example: dpdk_version="DPDK 21.11.2"
+  # metadata_fields = ["dpdk_pid"]
```

Comment:
Should these actually be fields?

Reply:
Yes, sorry. These values are fields, not tags, will change the description of PR

Reply:
No the question is if these really should be fields. As if they are representing metadata, it maybe fits better as being a tag?

Reply:
Initially field was a more convenient value to deliver these metrics.

Do you think changing these fields to tags will be more valuable?

Reply:
Yes, especially the `dpdk_version` as that truly is metadata. Does the PID change often? If it does and having it as a tag might introduce cardinality issues on some outputs.

Reply:
Field was selected mostly due to data cardinality - since PIDs could take various values - the number of different values could in time be too large to be used sensibly as a tag. 

Moreover there was a discussion with InfluxData folks about what should be treated a tag, and what a field. If I remember correctly it was a tag first, but we went with "safer" approach to avoid changing tags to fields after community review